NBitcoin/BitcoinCore/Coins.cs

```diff
@@ -9,29 +9,10 @@ namespace NBitcoin.BitcoinCore
 	public class Coins : IBitcoinSerializable
 	{
 		// whether transaction is a coinbase
-		bool fCoinBase;
-		public bool Coinbase
-		{
-			get
-			{
-				return fCoinBase;
-			}
-			set
-			{
-				fCoinBase = value;
-			}
-		}
+		public bool CoinBase { get; private set; }

```

Comment:
How do you know that no one is using the property setter? If someone is using the setter an you make it private then, you are breaking the other guy project. NBitcoin is a library and you never know how the developers are using it. Anyway, it is a small change and it is not uncommon to break something from time to time.